pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/SegmentMetadata.java

```diff
@@ -53,6 +53,10 @@ public interface SegmentMetadata {
 
   long getEndTime();
 
+  String getRawStartTime();
```

Comment:
Similarly we don't need these 2 methods. We can use the min/max value from the time column metadata